packages/tfjs-node-helpers/src/testing/metrics/tnr.ts

```diff
@@ -0,0 +1,14 @@
+import { ConfusionMatrix } from "../confusion-matrix";
+import { Metric } from "../metric";
+import { MetricCalculator } from "../metric-calculator";
+
+export class TNRMetricCalculator extends MetricCalculator {
```

Comment:
@VladimirGlinskii I think we can remove the `TNRMetricCalculator` and `TPRMetricCalculator `, because:

1. TNR is Specificity.
2. TPR is Recall.
3. We already provide calculators for Specificity and Recall.